src/main/java/org/jboss/forge/arquillian/command/core/ContainerSetupWizard.java

```diff
@@ -49,21 +55,26 @@
     private DependencyResolver resolver;
 
     @Inject
-    @Any
-    private Event<ContainerInstallEvent> installEvent;
-
-    @Inject
     @WithAttributes(shortName = 'f', label = "Container Adapter Type", type = InputType.DROPDOWN)
     private UISelectOne<ContainerType> containerAdapterType;
 
     @Inject
-    @WithAttributes(shortName = 'c', label = "Container Adapter", type = InputType.DROPDOWN, required = true)
+    @WithAttributes(shortName = 'a', label = "Container Adapter", type = InputType.DROPDOWN, required = true)
     private UISelectOne<Container> containerAdapter;
 
     @Inject
     @WithAttributes(shortName = 'x', label = "Container Adapter Version", type = InputType.DROPDOWN)
     private UISelectOne<String> containerAdapterVersion;
 
+
+    @Inject
+    @WithAttributes(shortName = 'i', label = "Do you want Arquillian to install the container?", type = InputType.CHECKBOX)
+    private UIInput<Boolean> installContainer;
+
+    @Inject
+    @WithAttributes(shortName = 'c', label = "Container Version", type = InputType.DROPDOWN)
```

Comment:
v is already taken in [ArquillianAddCommand](https://github.com/forge/arquillian-addon/blob/master/src/main/java/org/jboss/forge/arquillian/command/core/ArquillianAddCommand.java#L34), So we have conflict with this when using it with `arquillian-setup` command